templates/overrides/form/input--submit.html.twig

```diff
@@ -10,5 +10,13 @@
  * @see template_preprocess_input()
  */
 #}
-{# @TODO Revisit after refactoring the button pattern. #}
-<input{{ attributes.addClass(['btn', 'btn-primary']) }} />{{ children }}
+{% if attributes.hasClass('button--primary') %}
```

Comment:
in the login form, the class `button--primary` is not set for the login button, it seems that this class is added per use case and not automatically by a rule.
To solve this issue, we can do something like ``{% elseif attributes['data-drupal-selector'] == 'edit-submit' %}``

Reply:
Ok I'm going to add it as or condition in the same if. thanks.